### ISSUE TITLE
Val loss does not decrease

### ISSUE THREAD

**Issue by Vincent9797** [xuannianz/EfficientDet#185]
As seen in the screenshot below, train loss decreases but validation loss does not fall. Currently using TF 2.1.

Command used: `python train.py --snapshot imagenet --phi 0 --gpu 0 --random-transform --compute-val-loss --freeze-backbone --weighted-bifpn --batch-size 32 --steps 190 csv train.csv classes.csv --val val.csv`

![image](https://user-images.githubusercontent.com/42912553/85233155-eadff180-b436-11ea-8a27-76377c05f8c4.png)

I don't think there's issues with the anchors as I have ran debug.py from https://github.com/fizyr/keras-retinanet and it shows that the annotations do overlap with the anchors.

**Comment by sonfire186** [comment]
Increase the number of steps to 1000
